internal/srv/handlers.go

```diff
@@ -1,56 +1,61 @@
 package srv
 
 import (
-	"fmt"
+	"encoding/json"
 	"net/http"
-	"strings"
 
-	"github.com/Moby/Moby/pkg/namesgenerator"
 	"github.com/nats-io/nats.go"
+
+	events "go.infratographer.sh/loadbalanceroperator/pkg/events/v1alpha1"
+	"go.infratographer.sh/loadbalanceroperator/pkg/pubsubx"
+)
+
+const (
+	EventCreate = "create"
```

Comment:
should this come from the `events` package? Will we need it elsewhere?

Reply:
Yepp, I think this makes sense for now. Hoping to clean this up a bit more when we refactor some of this